src/draw/sdl/lv_draw_sdl.c

```diff
@@ -9,13 +9,13 @@
 #include "../lv_draw_private.h"
 #if LV_USE_DRAW_SDL
 #include LV_SDL_INCLUDE_PATH
-#include <SDL2/SDL_image.h>
```

Comment:
Please confirm it's remove as intended.

Reply:
Confirmed. It was one of the goals to get rid of `SDL_image`.